modules/lvms-provisioning-storage-using-logical-volume-manager-operator.adoc

```diff
@@ -8,6 +8,12 @@
 
 After you have created the LVM volume groups using the `LVMCluster` custom resource (CR), you can provision the storage by creating persistent volume claims (PVCs).
 
+The minimum storage size that you can request for each file system type is as follows:
```

Comment:
Just a wording suggestion. I was kind of looking at these guidelines https://www.ibm.com/docs/en/ibm-style?topic=format-lists#lead-in-wording

```suggestion
The following are the minimum storage sizes you can request for each file system type:
```

Reply:
Updated as per the suggestion.